website/docs/r/vapp_vm.html.markdown

```diff
@@ -240,8 +259,8 @@ The following arguments are supported:
 * `template_name` - (Optional; *v2.9+*) The name of the vApp Template to use
 * `vm_name_in_template` - (Optional; *v2.9+*) The name of the VM in vApp Template to use. For cases when vApp template has more than one VM.
 * `memory` - (Optional) The amount of RAM (in MB) to allocate to the VM. If `memory_hot_add_enabled` is true, then memory will be increased without VM power off.
-* `cpus` - (Optional) The number of virtual CPUs to allocate to the VM. Socket count is a result of: virtual logical processors/cores per socket. The default is 1. If `cpu_hot_add_enabled` is true, then cpus will be increased without VM power off.
-* `cpu_cores` - (Optional; *v2.1+*) The number of cores per socket. The default is 1
+* `cpus` - (Optional) The number of virtual CPUs to allocate to the VM. Socket count is a result of: virtual logical processors/cores per socket. If `cpu_hot_add_enabled` is true, then cpus will be increased without VM power off.
+* `cpu_cores` - (Optional; *v2.1+*) The number of cores per socket.
```

Comment:
Please elaborate, why did you remove the "The default is 1"?

Reply:
If you don't provide value, SDK will use defined default value. After Sizing policy added this isn't correct as these values can come from Sizing policy.

Reply:
> SDK will use defined default value

Defined where? Thanks.

Reply:
defined in schema. I removed this definition and as result remove here from docs.

Reply:
I'd say this deserves even a bug fix note in changelog. This was especially wrong for some people when they just want to run template with configured capacity and not explicitly defining it.

Reply:
added